examples/semantic_segmentation/README.md

```diff
@@ -54,16 +51,17 @@ Run a script using the `--help` flag for more information:
 
 ```shell
 $ poetry run python3 semantic_segmentation/seed_test_run.py --help
-usage: seed_test_run.py [-h] [--model MODEL]
-                        [--test_suites TEST_SUITES [TEST_SUITES ...]]
-                        --out_bucket OUT_BUCKET
+usage: seed_test_run.py [-h] [--eval_level EVAL_LEVEL] [--model MODEL]
+                        [--test_suites TEST_SUITES [TEST_SUITES ...]] --out_bucket OUT_BUCKET
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
+  --eval_level EVAL_LEVEL
+                        Number of points on precision/recall curve. Higher value will be more
+                        accurate, but require more compute
```

Comment:
My thought process was this value tunes the scripts bottleneck and is difficult to find otherwise. We can highlight the value with a comment, but I thought this was the better approach.